### ISSUE TITLE
ContentDialog Text Color Contrast

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#2354]
**Describe the bug**
The color contrast of hyperlinks and text in a ContentDialog does not meet the color contrast ratio of 4.5:1 for accessibility.

https://github.com/microsoft/terminal/issues/5395 is dependent on a fix for this issue.

**Steps to reproduce the bug**
This can be seen in Windows Terminal's [TerminalPage.xaml](https://github.com/microsoft/terminal/blob/master/src/cascadia/TerminalApp/TerminalPage.xaml) , but this just uses the built-in theme dictionaries.

1. Open your app of choice that uses a ContentDialog using `HyperlinkButton`s
2. Open Accessibility Insights ([download link here](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shcha1%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847087032&sdata=uWx6cTEfazn%2BoCV3cL1iYB4KN%2BuXQ/MTlmOVec%2BZgKc%3D&reserved=0)) to check the color contrast ratio
3. Verify whether the color contrast ratio for the SplitButton control meets the 4.5:1 contrast ratio

**Expected behavior**
In Windows Terminal, the links present in the "About" pop up window and the text to the back ground color in the "OK" button should meet the color contrast ratio 4.5:1

**Screenshots**
![image](https://user-images.githubusercontent.com/11050425/80646820-4935b880-8a22-11ea-9888-f5d7ceb9d8ef.png)

![image](https://user-images.githubusercontent.com/11050425/80646831-4d61d600-8a22-11ea-9baa-f279dc9e7fdc.png)

**Version Info**

NuGet package version: 

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| November 2019 Update (18363)  |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 |    |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Comment by ranjeshj** [comment]
@carlos-zamora  Is the blue color, the default blue shell accent color ?

**Comment by carlos-zamora** [comment]
> @carlos-zamora Is the blue color, the default blue shell accent color ?

Yes. This one was another one submitted by the accessibility testers. Personally, I feel that using the accent color is correct, but I get how that could create a situation like this where the color contrast is not acceptable.

CC'ing @DHowett-MSFT to keep him in the loop.

**Comment by mdtauk** [comment]
When choosing an Accent Color, it does warn where there could be visibility problems - so if the user has chosen such a colour - then we can presume they take that risk.

The Windows team should have chosen an accent colour that meets contrast ratio recommendations right?

**Comment by ranjeshj** [comment]
@chigy can you route this bug ?

**Comment by chigy** [comment]
We are aware of this issue and will be sure to share this as additional information to the team that owns addressing this issue.

**Comment by YuliKl** [comment]
This issue is now fixed in the OS ([by internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/26006362)). Corrected accent color will be the default in the next version of Windows.

**Comment by mdtauk** [comment]
> This issue is now fixed in the OS ([by internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/26006362)). Corrected accent color will be the default in the next version of Windows.

So this is the end of
# 00 78 D4

Looking at the design tokens, it looks like the way the Accent colour is applied to Light and Dark themes will be changing.  Any chance you could share some information about this new formula for picking colours?

**Comment by StephenLPeters** [comment]
@YuliKl ?
